src/OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -43,39 +43,38 @@ using orbit_grpc_protos::ProcessInfo;
 
 namespace capture_deserializer {
 
-ErrorMessageOr<CaptureListener::CaptureOutcome> Load(const std::string& file_name,
+ErrorMessageOr<CaptureListener::CaptureOutcome> Load(const std::filesystem::path& file_name,
                                                      CaptureListener* capture_listener,
                                                      ModuleManager* module_manager,
                                                      std::atomic<bool>* cancellation_requested) {
   SCOPED_TIMED_LOG("Loading capture from \"%s\"", file_name);
 
-  // Binary
-  std::ifstream file(file_name, std::ios::binary);
-  if (file.fail()) {
-    std::string error = absl::StrFormat("Error opening file \"%s\" for reading", file_name);
+  auto open_result = orbit_base::OpenFileForReading(file_name);
+  if (open_result.has_error()) {
+    std::string error = absl::StrFormat("Error opening file \"%s\" for reading: %s",
+                                        file_name.string(), open_result.error().message());
```

Comment:
If I remember correctly `OpenFileForReading` already wraps a generic message around SafeStrerror.

Reply:
Just checked. The message would look like this: 
```
Error opening file "filename.xyz" for reading: Unable to open file "filename.xyz": <the actual error from strerror>
```

I think it's fine to just return `open_result.error()`